org.lyx.LyX.Plugin.SweaveKnitr.yml

```diff
@@ -14,34 +14,24 @@ modules:
     buildsystem: autotools
     config-opts:
       - --disable-multilib
+      - --enable-multiarch
       - --enable-languages=c,c++,fortran
-      - --enable-__cxa_atexit
-      - --enable-checking=release
-      - --enable-clocale=gnu
+      - --disable-bootstrap
       - --enable-default-pie
       - --enable-default-ssp
-      - --enable-gnu-indirect-function
-      - --enable-gnu-unique-object
-      - --enable-install-libiberty
+      - --with-isl
+      - --disable-libssp
       - --enable-linker-build-id
-      - --enable-plugin
+      - --disable-libstdcxx-filesystem-ts
+      - --enable-cet
       - --enable-shared
-      - --enable-threads=posix
-      - --disable-bootstrap
-      - --disable-libssp
-      - --disable-libstdcxx-pch
-      - --disable-libunwind-exceptions
-      - --disable-multilib
-      - --disable-stage1-checking
-      - --disable-werror
-      - --with-linker-hash-style=gnu
-      - --with-system-zlib
 
     build-options:
       arch:
         x86_64:
           config-opts:
-            - --enable-libmpx
+            - --host=x86_64-unknown-linux-gnu
+            - --target=x86_64-unknown-linux-gnu
         arm:
```

Comment:
what about aarch64?

`arm` hasn't been supported for a few years.

Reply:
It is residual configuration.

It seems that the compiler did not compile.